README-oidc.md

```diff
@@ -0,0 +1,38 @@
+# Httpd Configmap Generator - OpenID-Connect (OIDC)
+
+This documents how to run the httpd\_configmap\_generator tool to configure the container against an OpenID-Connect (OIDC) identity provider.
+
+## Usage for the `oidc` auth-type:
+
+```
+$ httpd_configmap_generator oidc --help
+Options:
+  -o, --output=<s>                Configuration map file to create
+  -i, --oidc-url=<s>              OpenID-Connect Provider URL
+  -c, --oidc-client-id=<s>        OpenID-Connect Provider Client ID
+  -l, --oidc-client-secret=<s>    OpenID-Connect Provider Client Secret
```

Comment:
can these short options be -u for url, -i for id and -s for secret ?

Reply:
@abellotti Good catch! I thought I had fixed those. Will do!